index.js

```diff
@@ -0,0 +1,33 @@
+const Note = require('./lib/Notes');
+const Input = require('./lib/Input');
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+
+
+const input = new Input(process.argv);
+Input.valid(input) && Note.execute(input);
+
+
+mongoose.connect('mongodb://localhost:27017/notes', {
+  useNewUrlParser: true,
+  useUnifiedTopology: true
+});
+
+app.use(express.json());
+
+app.post('/notes', (req, res) => {
```

Comment:
nice. you didn't have to make an express server. The goal was to make a command line application, but cool!